host-provision.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo Command used:
+echo "$0 $@"
+echo
+
+shopt -s nocasematch
+
+# Check if we're using the Mac stock getopt and fail if true
+out=`getopt -T`
+if [ $? != 4 ]; then
+  echo -e "Your getopt does not support long parameters, possibly you're on a Mac, if so please install gnu-getopt with brew"
+  echo -e "\thttps://brewformulas.org/Gnu-getopt"
+  exit
+fi
+
+# set -x
+GETOPT_MANDATORY="inventory-file:"
+# GETOPT_OPTIONAL="ora-role-separation:,ora-disk-mgmt:,ora-swlib-path:,ora-staging:,ora-asm-disks:,ora-data-mounts:,help"
+GETOPT_OPTIONAL="hostprovision-ansible-user:,help"
+GETOPT_LONG="${GETOPT_MANDATORY},${GETOPT_OPTIONAL}"
+GETOPT_SHORT="h"
+
+
+INVENTORY_FILE="${INVENTORY_FILE:-./inventory_files/inventory}"
+
+
+options=$(getopt --longoptions "$GETOPT_LONG" --options "$GETOPT_SHORT" -- "$@")
+
+[ $? -eq 0 ] || {
+       echo "Invalid options provided: $*"
+       exit 1
+}
+
+# echo "PARSED COMMAND LINE FLAGS: $options"
+
+eval set -- "$options"
+
+while true; do
+    case "$1" in
+    --inventory-file)
+        INVENTORY_FILE="$2"
+        shift;
+        ;;
+    --hostprovision-ansible-user)
```

Comment:
From a user perspective, this naming is redundant;  we're asking for the user that bms-toolkit should run as.  Can we use the same parameter forms as isntall-oracle.sh has?

Reply:
Thanks for catching this!
Will change this to`--instance-ssh-user` keeping it consistent with from `install-oracle.sh`